pages/dkp/konvoy/2.2/choose-infrastructure/vsphere/new/index.md

```diff
@@ -50,7 +50,7 @@ Before you begin, make sure you have created a [Bootstrap][bootstrap] cluster.
       --data-store <DATASTORE_NAME> \
       --folder <FOLDER_NAME> \
       --server <VCENTER_API_SERVER_UTR \
-      --ssh-authorized-key "<SSH_KEY>" \
+      --ssh-public-key-file "<SSH_PUBLIC_KEY_FILE>" \
```

Comment:
Good catch.

Oh, but do we need the quotes in there? I think just `<SSH_PUBLIC_KEY_FILE>` is right

```suggestion
      --ssh-public-key-file "<SSH_PUBLIC_KEY_FILE>" \
```